docs/articles/documentation/using-testcafe/common-concepts/browser-support.md

```diff
@@ -29,33 +33,38 @@ Mozilla Firefox   | 17+    | `firefox`
+TestCafe can run tests in browsers on remote devices. The only requirement is a device should have network access to the TestCafe server.
 
-If you are working with TestCafe through the command line,
-[list the desired browsers](../command-line-interface.md#browser-list)
-within the `testcafe` command.
+At first, you need to create a remote browser connection. You can do this from CLI by specifying the `remote` *alias* or from API by using the [createBrowserConnection](../programming-interface/testcafe.html#createbrowserconnection) method.
 
-In case you are using the API, specify the browsers
-with the [runner.browsers](../programming-interface/runner.md#browsers) method.
+After that, TestCafe will provide URLs that you need to open in required browsers on your remote device to start testing.
+
+## Cloud Browsers
+
+TestCafe allows you to use cloud browsers that can be accessed through *browser provider plugins*.
 
-To identify browsers, different approaches are used.
+For example, there are browser provider plugins for testing in [Sauce Labs](https://saucelabs.com/) and [PhantomJS](http://phantomjs.org/) browsers:
+
+* [testcafe-browser-provider-saucelabs](https://www.npmjs.com/package/testcafe-browser-provider-saucelabs)
+* [testcafe-browser-provider-phantomjs](https://www.npmjs.com/package/testcafe-browser-provider-phantomjs)
+
+If you need to use a different cloud browser, create a custom [browser provider plugin](../../extending-testcafe/browser-provider-plugin/).
+
+> Created browser provider packages can be published to npm. The provider package name consists of two parts - the `testcafe-browser-provider-` prefix and the name of a provider itself; e.g., `testcafe-browser-provider-saucelabs`.
+So, you can search for available browser provider packages on npm by the `testcafe-browser-provider-` prefix: [https://www.npmjs.com/search?q=testcafe-browser-provider](https://www.npmjs.com/search?q=testcafe-browser-provider).
 
-Browser&#160;Type  | Specified by
------------------- | ----------------------------------
-Local browser      | The path to the browser executable or browser short name. For the full list of names, see the table in the [Directly Supported Browsers](#directly-supported-browsers) section. To obtain the list of browsers installed on the current machine, run TestCafe from the command line with the [-b option](../command-line-interface.md#-b---list-browsers).
-Browser&#160;accessed&#160;through a&#160;[browser&#160;provider&#160;plugin](../common-concepts/browser-support.md#browser-provider-plugins). | The browser alias that consists of the browser provider name and the name of a browser itself.
-Remote browser     | The `remote` alias or [BrowserConnection](../programming-interface/browserconnection.md).
\ No newline at end of file
+To specify a cloud browser for a test task, use a *browser alias* that consists of the browser provider name and the name of a browser itself; for example, `saucelabs:Chrome@52.0:Windows 8.1`.
```

Comment:
It's not correct for all providers, some providers can have browser name omitted, e.g. `phantomjs`. The better way to obtain available aliases for provider is to run `testcafe --list-browsers {shortProviderName}`, there `shortProviderName` is what follows after `testcafe-browser-provider-` prefix, e.g. `saucelabs` in `testcafe-browser-provider-saucelabs`